### PR TITLE
[CIS-729] Remove setting of TAMIC inside pin methods

### PR DESCRIPTION
**Pull request by DominikBucher12**
# What this PR do:
- Removes TAMIC from pin view extensions and replaces the 
# How to test it
- Probably CI can take care of UI Tests, if they pass, we are fine :) 
# Where you can start
- `UIView+Extensions.swift`

### CODE REVIEW

File: Sources/StreamChatUI/CommonViews/AvatarView/ChatChannelAvatarView+SwiftUI.swift [hunk 1]
```diff
@@ -39,6 +39,7 @@ extension _ChatChannelAvatarView {
         }
 
         override public func setUpLayout() {
+            hostingController?.view.translatesAutoresizingMaskIntoConstraints = false
             embed(hostingController!.view)
```
Comment (VojtaStavik) [inline Sources/StreamChatUI/CommonViews/AvatarView/ChatChannelAvatarView+SwiftUI.swift:43]: I searched our codabase for `embed(hostingController!.view)` and it found 3 cases. Why do we change it just for this one?

**Review by VojtaStavik — APPROVED** [review]
LGTM, however, just a simple text search found places where the value is not set correctly. Can you please double-check and make sure we **really** set the value where needed?

File: Sources/StreamChatUI/CommonViews/AvatarView/ChatChannelAvatarView+SwiftUI.swift (hunk 1 above)
Comment (DominikBucher12) [reply Sources/StreamChatUI/CommonViews/AvatarView/ChatChannelAvatarView+SwiftUI.swift:43]: Maybe the others are not tested yet :) I will check, thank you. :)

Comment (DominikBucher12) [reply Sources/StreamChatUI/CommonViews/AvatarView/ChatChannelAvatarView+SwiftUI.swift]: Wow very thanks for this inside, I actually overlooked that `TAMIC = false` is being set in tests for SwiftUI views, which was part of strange fails when I was trying to figure out. `HostingVC.view` doesn't need to have this property set to false... I deleted the setting from tests and now we should be fine. :)

**Review by b-onc — APPROVED** [review]
:shipit:

**Pull request merged**